documentation/modules/exploit/multi/http/wp_plugin_fma_shortcode_unauth_rce.md

```diff
@@ -0,0 +1,281 @@
+## Vulnerable Application
+
+WordPress File Manager Advanced Shortcode 2.3.2 - Unauthenticated Remote Code Execution through shortcode.
+
+The WordPress plugin does not adequately prevent uploading files with disallowed MIME types when using the shortcode.
+This leads to RCE in cases where the allowed MIME type list does not include PHP files.
+In the worst case, this is available to unauthenticated users, but is also works in an authenticated configuration.
+
+File Manager Advanced Shortcode plugin version `2.3.2` and lower are vulnerable.
+To install the Shortcode plugin, File Manager Advanced version `5.0.5` or lower is required to keep the configuration vulnerable.
+Any user can exploit this vulnerability which results in access to the underlying operating system with the same privileges
+under which the WordPress web services run.
+
+For more information, see [This Article](https://attackerkb.com/topics/JncRCWZ5xm/cve-2023-2068).
+
+This module has been tested on:
+* Windows  Server 2019 Standard and Kali Linux running on Raspberry PI.
+* WordPress 6.2.2
+* File Manager Advanced 5.0.5
+* File Manager Advanced Shortcode 2.3.2
+
+**Instructions for a vulnerable WordPress installation:**
+For Windows and Linux follow these instructions to install and configure WAMP64 and WordPress:
+[Install WordPress locally](https://www.hostinger.com/tutorials/install-wordpress-locally)
+
+After you have successfully installed and configures WordPress, follow the below steps to install the vulnerable plugins and configure
+a web page with the file-manager-advanced shortcode embedded.
+1. Download WordPress plugins:
+* [File Manager Advanced 5.0.5](https://github.com/h00die-gr3y/Metasploit/blob/main/images/file-manager-advanced.zip)
+* [File Manager Advanced Shortcode 2.3.2](https://github.com/h00die-gr3y/Metasploit/blob/main/images/file-manager-advanced-shortcode-2.3.2-mdnhux.zip)
```

Comment:
To the tester, let's see if we can implement alternative steps here for test lab setup.

i.e. Docker for wordpress, and potentially archived links for older versions of the plugins that are available from wordpress directly

Reply:
@adfoster-r7, any suggestions?
I have a hard time finding these archive links on the web.
The `WordPress` Docker is not that hard (`docker pull wordpress`), but the plugins are pretty difficult, especially the `shortcode` plugin.